_src/static/sass/main.scss

```diff
@@ -0,0 +1,66 @@
+// ------------------------------
```

Comment:
Is this file new? Previously there two main compile files created for LTR and RTL (``main-ltr.scss`` and ``main-rtl.scss``). This file should not exist as we need those two separate files in its place.